VirtualEconomyFramework/VENFTApp-Blazor/Components/Account/TransactionDetails.razor

```diff
@@ -315,7 +318,7 @@
         StateHasChanged();
         showNFTDetailsVisible = true;
     }
-    private async Task showNFTDetailsCancel(MouseEventArgs e)
+    private void showNFTDetailsCancel()
     {
         showNFTDetailsVisible = false;
     }

```

Comment:
I don't think we somewhere use `MouseEventArgs`, so I removed them. I also changed return type to `void` instead of `Task`, because it not needed.

Reply:
Good point. no we do not using it.